src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormImages.tsx

```diff
@@ -19,8 +19,9 @@ export interface MyCollectionArtworkFormImagesProps {
   saveImagesToLocalStorage: (artworkId: string) => Promise<string | undefined>
 }
 export const MyCollectionArtworkFormImages = forwardRef<
-  MyCollectionArtworkFormImagesProps
->((_, formImagesRef) => {
+  MyCollectionArtworkFormImagesProps,
+  { isEditing?: boolean }
+>(({ isEditing = false }, formImagesRef) => {
```

Comment:
**suggestion:** Is this prop really needed? When editing an artwork, there shouldn't be any images from already existing artworks, right? I'd suggest to remove this prop if it's not needed to reduce complexity.

Reply:
my goal here was to separate between the following:
- images that are added from an existing artwork
- images that are added from the same artwork while editing it (they source also contains `cloudFront`)

Both images urls contain `cloudFront`

I wasn't able to separate between them without explicitly sending the flag

Reply:
Oh, I understand 👍